wp-seo-main.php

```diff
@@ -11,7 +11,7 @@
 }
 
 /**
- * @internal Nobody should be able to overrule the real version number as this can cause serious issues
+ * @note Nobody should be able to overrule the real version number as this can cause serious issues
```

Comment:
@maartenba Thanks for your feedback.

> Well, @internal means nobody outside the project or module should call this function.

Not *just* that according to the docs:
> The @internal tag is used to denote that associated Structural Elements are elements internal to this application or library. **It may also be used inside a long description to insert a piece of text that is only applicable for the developers of this software.**